qa/automation/CareerPortal.Tests.Scenarios/1. Search by Filters/1.1. Search by Filters.feature

```diff
@@ -0,0 +1,154 @@
+    And User should see default page with positions 
+
+Scenario: 1.1_09 Checking the form "Open job positions" with filters
+    Given https://career.quantori.com/ru/positions website is opened 
+    When User opened competence list box filter 
+    And User Chooses Databases competence 
+    And User opened technologies list box filter 
+    And User Chooses SQL technology
+    And User pushes "Open job positions" form
+    Then User should be redirected to the new page
+    And  User should see default page "Quantori open job positions" without filters
+
+Scenario: 1.1_10 Saving the cache when the page is refreshed on main Quantori page
+    Given https://career.quantori.com website is opened 
+    When User opened competence list box filter 
+    And User Chooses Databases competence 
+    And User opened technologies list box filter 
+    And User Chooses SQL technology
+    And User refreshes page
+    Then User should see default page with filters
+
+Scenario: 1.1_11 Saving the cache when the page is refreshed on "Quantori open job positions" page
+    Given  https://career.quantori.com website is opened 
+    When User opened competence list box filter 
+    And User Chooses Databases competence 
+    And User opened technologies list box filter 
+    And User Chooses SQL technology
+    And User refreshes page
+    Then User should see https://career.quantori.com/ru/positions?competences=Databases&technologies=SQL page with filters
+
+Scenario: 1.1_12  Checking the checkboxes when selecting a competency
+    Given https://career.quantori.com website is opened 
+    When User opened competence list box filter 
+    And User Chooses Databases competence
+    Then User should see activated checkbox
+
+Scenario: 1.1_13  Checking the checkboxes when selecting a technology
+    Given https://career.quantori.com website is opened 
+    When User opened technology list box filter 
+    And User Chooses SQL technology
+    Then User should see activated checkbox
+
+Scenario: 1.1_14  Checking the competency counter change when selecting the checkbox in the filter
+    Given https://career.quantori.com website is opened 
+    When User opened competence list box filter 
+    And User Chooses Databases competence
+    Then User should see the competence counter change from empty to 1
+
+Scenario: 1.1_15  Checking the technology counter change when selecting the checkbox in the filter
+    Given https://career.quantori.com website is opened 
+    When User opened technology list box filter 
+    And User Chooses SQL technology
+    Then User should see the technology counter change from empty to 1
+
+Scenario: 1.1_16 Checking the selection of incompatible checkboxes
+    Given https://career.quantori.com/ru/positions website is opened 
+    When User opened competence list box filter
+    And User Chooses Business Analysis  competence
+    And User opened technology list box filter
+    And User Choose .NET technology
```

Comment:
When User chooses Business Analysis competence with list box filter
And User chooses .NET technology with list box filter

Reply:
corrected